common/metrics/metric_defs.go

```diff
@@ -1232,17 +1232,20 @@ var (
 	PersistenceSessionRefreshAttempts      = NewCounterDef("persistence_session_refresh_attempts")
 
 	// Common service base metrics
-	RestartCount         = NewCounterDef("restarts")
-	NumGoRoutinesGauge   = NewGaugeDef("num_goroutines")
-	GoMaxProcsGauge      = NewGaugeDef("gomaxprocs")
-	MemoryAllocatedGauge = NewGaugeDef("memory_allocated")
-	MemoryHeapGauge      = NewGaugeDef("memory_heap")
-	MemoryHeapIdleGauge  = NewGaugeDef("memory_heapidle")
-	MemoryHeapInuseGauge = NewGaugeDef("memory_heapinuse")
-	MemoryStackGauge     = NewGaugeDef("memory_stack")
-	NumGCCounter         = NewBytesHistogramDef("memory_num_gc")
-	GcPauseMsTimer       = NewTimerDef("memory_gc_pause_ms")
-	NumGCGauge           = NewGaugeDef("memory_num_gc_last",
+	RestartCount           = NewCounterDef("restarts")
+	NumGoRoutinesGauge     = NewGaugeDef("num_goroutines")
+	GoMaxProcsGauge        = NewGaugeDef("gomaxprocs")
+	MemoryAllocatedGauge   = NewGaugeDef("memory_allocated")
+	MemoryHeapGauge        = NewGaugeDef("memory_heap")
+	MemoryHeapObjectsGauge = NewGaugeDef("memory_heap_objects")
+	MemoryHeapIdleGauge    = NewGaugeDef("memory_heapidle")
+	MemoryHeapInuseGauge   = NewGaugeDef("memory_heapinuse")
+	MemoryStackGauge       = NewGaugeDef("memory_stack")
+	MemoryMallocs          = NewGaugeDef("memory_mallocs")
+	MemoryFrees            = NewGaugeDef("memory_frees")
```

Comment:
nit: rename to be consistent?
```suggestion
	MemoryMallocsGauge     = NewGaugeDef("memory_mallocs")
	MemoryFreesGauge       = NewGaugeDef("memory_frees")
```